apps/paymaster-proxy/package.json

```diff
@@ -10,7 +10,11 @@
     "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\"",
     "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix --quiet && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn",
     "generate-example-env": "ts-node scripts/generateExampleEnv.ts",
-    "run:loadtest": "pnpm dlx artillery run loadtest/basic-loadtest.yml"
+    "run:loadtest": "pnpm dlx artillery run loadtest/basic-loadtest.yml",
+    "docker:services:up": "docker compose up -d redis",
+    "docker:up": "docker compose up -d redis paymaster-proxy-app",
+    "docker:replicas:up": "docker compose up -d redis paymaster-proxy-app-replica reverse-proxy",
+    "docker:down": "docker-compose down"
```

Comment:
this should probably be `docker compose down`, we should avoid using `docker-compose` since this is the old way

Reply:
good point - sry forgot to push this change, will get merged in with a future PR